libs/designer/src/lib/core/utils/parameters/__test__/helper.spec.ts

```diff
@@ -1,131 +1,530 @@
-import { parameterValueToJSONString, parameterValueToString } from '../helper';
-import type { ParameterInfo, ValueSegment } from '@microsoft/designer-ui';
+import { getParameterEditorProps, parameterValueToJSONString, parameterValueToString } from '../helper';
```

Comment:
This diff is terrible&mdash;all the existing code is unchanged except that it's wrapped in a new `describe` block.